template/src/pages/home/Home.module.css

```diff
@@ -32,12 +38,23 @@
   line-height: 24px;
   text-align: center;
   word-wrap: break-word;
+
+  /* stylelint-disable-next-line media-feature-range-notation */
+  @media (max-width: 768px) {
```

Comment:
but yeah, I never seen media query written in the way you do. I alwasy so them with `>=`